packages/blocks/countdown/src/date-picker-input.tsx

```diff
@@ -41,7 +41,7 @@ const CustomInput = forwardRef<
         data-expanded={displayTime ? "true" : "false"}
         type="text"
         ref={(element) => {
-          if (!ref || !inputRef) return;
+          if (!ref) return;
```

Comment:
We should look at using `useForkRef` for this at some point https://react-hooks.org/docs/useForkRef